package-lock.json

```diff
@@ -2915,6 +2915,211 @@
         "esutils": "^2.0.2"
       }
     },
+    "@babel/preset-typescript": {
```

Comment:
I was surprised about this PR adds this but I checked in `develop` and looks like the `package-lock.json` is out-dated. I ran `npm install` in the latest commit on `develop` and the file was modified 🤷‍♂️ .